charts/rode-ui/values.yaml

```diff
@@ -8,6 +8,32 @@ rode:
   # App default is http://localhost:50052
   url: http://rode:50052
 
+  ## Rode authentication settings
+  ## Use rode.auth.oidc.enabled to enable OIDC auth
+  ##
+  auth:
+    oidc:
+      enabled: false
```

Comment:
We'll want to be able to set scopes here as well, but it can be a follow-up since our Keycloak config doesn't require it.

Reply:
We'll followup with the scopes for rode-ui.  for consistency, I might change the env var rode-ui expects from `OIDC_SCOPE` to `OIDC_SCOPES`, since that's the one that all of the go codebases is looking for